drivers/media/platform/virtio/virtio-camera.c

```diff
@@ -570,6 +570,180 @@ static int vcam_s_input(struct file *filp, void *p, unsigned int i)
 	return 0;
 }
 
+static int vcam_g_ctrl(struct file *file, void *fh, struct v4l2_control *control)
+{
+    struct virtio_camera_video *vnode = video_drvdata(file);
+    struct virtio_camera_ctrl_req *vcam_req;
+    int err;
+
+    vcam_req = virtio_camera_create_req(VIRTIO_CAMERA_CMD_GET_CTRL);
+    if (unlikely(vcam_req == NULL)) {
+        pr_err("virtio-camera: vnode%d fail to init get_ctrl-req, no mem.\n", vnode->idx);
+        return -ENOMEM;
+    }
+
+    vcam_req->ctrl.u.ctrl.id = control->id;
+
+    err = vcam_vq_request(vnode, vcam_req, NULL, 0, false);
+    if (err) {
+        pr_err("virtio-camera: vnode%d get_ctrl failed, err response.\n", vnode->idx);
+        goto err_free;
+    }
+
+    control->value = vcam_req->resp.u.ctrl.value;
+
+err_free:
+    kfree(vcam_req);
+    return err;
+}
+
+#include <linux/videodev2.h>
```

Comment:
Why not move include <>  line to the top of this file.

Reply:
Thanks, I have updated it.